libs/wallet-ui/src/__mocks__/service.ts

```diff
@@ -18,21 +18,23 @@ export const service: Service = {
       url: 'https://github.com/vegaprotocol/vegawallet-ui',
     }),
   GetVersion: () =>
-    Promise.resolve({
-      version: '0.98.0',
-      gitHash: '0x0',
-      networksCompatibility: [
-        {
-          network: 'test',
-          isCompatible: false,
-          retrievedVersion: '1.0.0',
-        },
-      ],
-      backend: {
-        version: '2.0.1',
+    Promise.resolve(
+      storageMock('service.GetVersion', {
```

Comment:
Can we throw these method names into an enum, this is repeated 4 times and if it changed it would be annoying to refactor 👍